src/components/AssetDetails/AssetVerifiedComponent.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Icon, Table } from 'semantic-ui-react';
+
+import DropdownComponent from '../common/DropdownComponent';
+
+class AssetVerifiedComponent extends Component {
+  state = {
+    show: false,
+    isChecked: this.props.assetDetail.verified,
+    optionText: this.props.assetDetail.verified ? 'Yes' : 'No'
```

Comment:
This does not have to be in state. You can derive the text in the render function. I.e.
```jsx
render() {
  const text = this.props.assetDetail.verified ? 'Yes' : 'No';
  ...
}
```